README.md

```diff
@@ -1,2 +1,232 @@
-# elixir-style-guide
-Our Elixir style guide
+<img src="https://jungsoft.io/static/media/jungsoft_logo.c44eaf52.png" width="300px"/>
+
+# Elixir Style Guide
+
+Welcome to Jungsoft's Elixir Style Guide. Sit down, have a cup of coffee and read this calmly. ☕
+
+## Indroduction
+
+The goal of this guide is to help our team to understand and
+follow code style best practices, maintaining a pattern.
+
+As this guide is an extension of the [Christopher Adams Elixir Style Guide](https://github.com/christopheradams/elixir_style_guide), we **highly recommend** reading it before you continue.
+
+# Table of Contents
+
+* [Alias Organization](#alias-organization)
+* ['With do' Organization](#with-do-organization)
+* [Maps](#maps)
+* [Lists](#lists)
+
+## Aliases
+
+### Comma
```

Comment:
We're kind of repeating the same rule everywhere, maybe it should be a separate rule, called "Dangling commas" where we say that all enumerables that span multiple lines should have dangling commas. That means:

- aliases
- keyword lists
- lists
- maps